packages/bridge-controller/package.json

```diff
@@ -54,7 +54,7 @@
     "@ethersproject/contracts": "^5.7.0",
     "@ethersproject/providers": "^5.7.0",
     "@metamask/accounts-controller": "^35.0.0",
-    "@metamask/assets-controllers": "^91.0.0",
+    "@metamask/assets-controllers": "^92.0.0",
```

Comment:
To discuss, now that these have been moved into dependencies, does this mean we are responsible for bumping other teams packages and/or updating their changelogs?